### PR TITLE
Feature/add devcontainer

### PR DESCRIPTION
**Pull request by cccs-tom**

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.177.0/containers/java
+{
+	"name": "Trino",
+	"build": {
+		"dockerfile": "Dockerfile",
+		"args": {
+			// Update the VARIANT arg to pick a Java version: 11, 15
+			"VARIANT": "11",
```
Comment (cccs-nik) [inline .devcontainer/devcontainer.json:9]: Not sure if it matters but it's 11 here but 15 in the dockerfile.

File: .devcontainer/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.177.0/containers/java/.devcontainer/base.Dockerfile
+
+# [Choice] Java version: 11, 15
+ARG VARIANT="15"
+FROM mcr.microsoft.com/vscode/devcontainers/java:0-${VARIANT}
+
+# [Option] Install zsh
+ARG INSTALL_ZSH="true"
```
Comment (dparent1) [inline .devcontainer/Dockerfile:8]: I know you override it but perhaps we should always set this to false by default in the Dockerfile too.  I think I need to make that change for my superset devcontainer as well.

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (dparent1) [reply .devcontainer/devcontainer.json:9]: I believe 11 will override the default 15 in the Dockerfile.  Can't wait for 17 to be an option as well...

**Review by dparent1 — APPROVED** [review]
Looks good and I know I've gotten the trino devcontainer running in the past.  You can change the Dockerfile INSTALL_ZSH if you want but I won't hold up the review for it.

File: .devcontainer/devcontainer.json (hunk 1 above)
Comment (cccs-tom) [reply .devcontainer/devcontainer.json:9]: Correct. The 15 in the Dockerfile is the default value (carried over from the devcontainer I copied at the start). This 11 overrides it and is what we want for Trino.

File: .devcontainer/Dockerfile (hunk 2 above)
Comment (cccs-tom) [reply .devcontainer/Dockerfile:8]: Sure, I don't mind. The reason is the same as in my previous comment. It's a carry-over and just a default. It only matters if it's not specified in the build args.

**Review by cccs-mmrouet — APPROVED** [review]

**Review by cccs-nik — APPROVED** [review]

**Pull request merged**